msvc/strings.h

```diff
@@ -0,0 +1,5 @@
+/* MSVC does not have strings.h */
+#ifndef _INC_STRINGS_H
+#define _INC_STRINGS_H 1
+#include <string.h>
+#endif
```

Comment:
Newline missing. GitHub UI shows that. I would expect some of our checks fails with this.

Reply:
`sed -i '$a\' msvc/strings.h`